index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Portfolio</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="style.css">
+    </head>
+
+    <body>
+        <header>
+            <nav class="nav_container">
+                <div class="logo">
+                     <img src="images/MyLogo.png" alt="Logo">
+                </div>
+     
+                <div class="toggle">
+                    <img src="images/Union.png" alt="Logo">
+                </div>
+             </nav>
+        </header>
+
+        <main>
+            <div class="main_container">
+               <div class="box1">
+                   I'm Alex Glad to see you!
+               </div>
+               <div class="box2">
+                I'm a software developer! I can help you build a product, 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, don't hestiate to contact me.
+               </div>
```

Comment:
- [ ] Please consider using `p` tag here instead of a `div` tag as this is a paragraph text. This helps to improve the semantic structure of your  HTML. See my suggestion below:

```suggestion
               <p class="box2">
                I'm a software developer! I can help you build a product, 
                feature or website Look through some of my work and experience! 
                If you like what you see and have a project you need coded, don't hestiate to contact me.
               </p>
```